expression-language/5.0/_index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Jakarta Expression Language 5.0 (under development)"
+date: 2021-04-15
+summary: "Release for Jakarta EE 10"
+---
+Jakarta Expression Language defines an expression language for Java applications.
+
+* [Jakarta Expression Language 5.0 Release Record](https://projects.eclipse.org/projects/ee4j.el/releases/5.0.0)
+    * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
+* [Jakarta Expression Language 5.0 Specification Document](./jakarta-expression-language-spec-4.0.pdf) (PDF)
+* [Jakarta Expression Language 5.0 Specification Document](./jakarta-expression-language-spec-4.0.html) (HTML)
+* [Jakarta Expression Language 5.0 Javadoc](./apidocs)
+* [Jakarta Expression Language 5.0 TCK](https://download.eclipse.org/jakartaee/expression-language/5.0/jakarta-expression-language-tck-5.0.0.zip)([sig](https://download.eclipse.org/jakartaee/expression-language/5.0/jakarta-expression-language-tck-5.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/expression-language/5.0/jakarta-expression-language-tck-5.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+* Maven coordinates
+     * [jakarta.el:jakarta.el-api:jar:5.0.0](https://repo1.maven.org/maven2/jakarta/el/jakarta.el-api/5.0.0/)
```

Comment:
I'm not sure you need these bits yet - it's just a plan review at this stage.
(You're also pointing at the 4.0 spec!)

Can you remove them please?

Reply:
Done.